analysis_options.yaml

```diff
@@ -21,6 +21,7 @@
 analyzer:
   strong-mode:
     implicit-dynamic: false
+#    implicit-casts: false
```

Comment:
Can you add a comment explaining why this is commented out?

Reply:
I shouldn't have checked that in. Removed.